src/main/java/duke/command/TodoCommand.java

```diff
@@ -0,0 +1,49 @@
+package duke.command;
+
+import duke.exception.DukeException;
+import duke.initials.Todo;
+import duke.storage.Storage;
+import duke.tasklist.TaskList;
+import duke.ui.Ui;
+
+import java.text.SimpleDateFormat;
+
+public class TodoCommand extends Command {
+
+    public static final SimpleDateFormat DATE_FORMAT = new SimpleDateFormat("dd/MM/yyyy HHmm");
```

Comment:
i think this is unused for a todo command

Reply:
oh yes will delete